CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
+- Use native logger backend instead of the Android instance directly. [#5520](https://github.com/mapbox/mapbox-navigation-android/pull/5520)
```

Comment:
qq: changes are looked like internal, do we want to add them to the changelog. They are not affecting customers

Reply:
Thought about it and as we're deprecating the `Logger` from `ReplayRouteMapper` wanted to include it. In any case, we can decide when releasing. cc @Zayankovsky